llpc/test/shaderdb/PipelineTess_TestInOutPacking.pipe

```diff
@@ -0,0 +1,172 @@
+; BEGIN_SHADERTEST
+; RUN: amdllpc -spvgen-dir=%spvgendir% -v %gfxip %s | FileCheck -check-prefix=SHADERTEST %s
+; SHADERTEST-LABEL:.AMDGPU.disasm
+; SHADERTEST-LABEL:_amdgpu_ls_main
+; SHADERTEST: ds_write2_b32 v0, v6, v7 offset0:2 offset1:3
+; SHADERTEST: ds_write2_b32 v0, v6, v7 offset1:1
+; SHADERTEST-LABEL:_amdgpu_hs_main
+; SHADERTEST: ds_read2_b32 v[0:1], v2 offset0:2 offset1:3
+; SHADERTEST: ds_read2_b32 v[2:3], v2 offset1:1
+; SHADERTEST-LABEL:_amdgpu_vs_main
```

Comment:
I am not sure which symbol name is used when ngg is enabled on gfx10+, please double check it.
ps: It is better to check the llvm-ir  after patch phase,  since we can't control the final ISA code completely.

Reply:
Ok. I will check llvm-ir.